spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -109,10 +109,12 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
       "spark.comet.cast.stringToTimestamp is disabled")
   }
 
-  ignore("cast string to timestamp") {
-    withSQLConf((CometConf.COMET_CAST_STRING_TO_TIMESTAMP.key, "true")) {
-      val values = Seq("2020-01-01T12:34:56.123456", "T2") ++ generateStrings(timestampPattern, 8)
-      castTest(values.toDF("a"), DataTypes.TimestampType)
+  test("cast string to timestamp") {
+    withSQLConf(
+      (SQLConf.SESSION_LOCAL_TIMEZONE.key -> "UTC"),
```

Comment:
If I change this to a different timezone then the test fails. Perhaps we can fall back to Spark for now if the timezone is not UTC?

Reply:
@andygrove I would need some help here. Do you want me to add a check here?

https://github.com/apache/datafusion-comet/blob/c90081996c3b0537695f1fd68bdef9aa22e08a68/spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala#L578

Reply:
This is done, we now fall back to spark if timezone is not "UTC"